googlemock/docs/cook_book.md

```diff
@@ -229,7 +229,7 @@ If a mock method has no `EXPECT_CALL` spec but is called, Google Mock
 will print a warning about the "uninteresting call". The rationale is:
 
   * New methods may be added to an interface after a test is written. We shouldn't fail a test just because a method it doesn't know about is called.
-  * However, this may also mean there's a bug in the test, so Google Mock shouldn't be silent either. (Note that the user should [*not* add an `EXPECT_CALL()`](https://github.com/google/googletest/blob/master/googlemock/docs/CookBook.md#knowing-when-to-expect) to suppress the warning, even if they think the call is harmless).
+  * However, this may also mean there's a bug in the test, so Google Mock shouldn't be silent either. (Note that the user should [*not* add an `EXPECT_CALL()`](#knowing-when-to-expect) to suppress the warning, even if they think the call is harmless).
```

Comment:
```suggestion
  * However, this may also mean there's a bug in the test, so Google Mock shouldn't be silent either. (Note that the user should [*not* add an `EXPECT_CALL()`](https://github.com/google/googletest/blob/master/googlemock/docs/cook_book.md#knowing-when-to-expect) to suppress the warning, even if they think the call is harmless).
```

Reply:
We need full path for internal/external sync reasons, at least for now

Reply:
Sure thing, will do. I'd rather not introduce an unnecessary commit, so I'll just push force the improved version. Hope you don't mind it.